app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java

```diff
@@ -68,7 +68,7 @@ private static void setImageUrlWithSize(ImageView imageView, @Nullable String im
             final int size = Math.round(sizeInDimen);
             imageView.setBackground(ContextCompat.getDrawable(imageView.getContext(), R.drawable.circle_border_grey200));
             Picasso.with(imageView.getContext())
-                    .load(convertToFullImageUrl(imageUrl))
```

Comment:
This image url should be converted in ViewModel class.
Plus this method is called from `setPhotoImageUrl()`, so it's not good to convert here.